### ISSUE TITLE
What version of Ubuntu and Node are supported?

### ISSUE THREAD

**Issue by nostyleguy** [pixelnest/presskit.html#2]
Hi. I had a been using your presskit for a while without issues, until I tried to update to presskit 0.6.1. Now whenever I try to 'presskit build' I get:

`Finding data...
There was an error during the parsing of `/root/presskit/data.xml`. Check that your XML document is valid.
You can use a validator like this: http://codebeautify.org/xmlvalidator
`
This is on Ubuntu 16.04 LTS with Node 6.10.1

I tried making an entirely fresh presskit with 

`presskit new <dir>` 

However, I still get the parse error even though the data.xml there is the default.

I assume this has to do with Node (I'm still not sure which of the many ways is the best to install Node), so I wanted a completely new testing environment. Here is what I did on a separate machine:

Install Ubuntu 16.04.3 LTS
`sudo apt install nodejs`
sudo npm install -g presskit`

This gives me tons of warnings because apparently the version of Node that comes with Ubuntu 16.04 is only 4.2? If I try to run presskit after this point, I get javascript erros in the actual presskit generator. 

at this point, I uninstalled the nodejs that is maintained via Ubuntu, and manually installed node 6.11.3 which is the most current. 

After uninstalling an reinstalling presskit, I'm back to the original xml parsing error. (I tried both 0.6.1 and 0.5.0)

tl;dr: What version of Ubuntu and Node actually works with presskit?

**Comment by valryon** [comment]
Hello,
It seems related to the other issue (https://github.com/pixelnest/presskit.html/issues/3). 
Looks like one of the plugin we're using was the source of the error. We're working on a patch. :)

Cheers.

**Comment by solarsailer** [comment]
You need Node 7.6 at least.

For the XML problem, it should be fixed with latest `0.6.2`. Update your `package.json` and re-open the issue if the problem is still here.

Thanks!

**Comment by nostyleguy** [comment]
Thanks for the quick response! 

Unfortunately, I am still getting the same xml parsing error with the following environment:

Ubuntu: 16.04 LTS
Node: 8.5.0
Presskit: 0.6.3

Note: I don't think I have permission to re-open this issue.

**Comment by solarsailer** [comment]
I'm able to reproduce it on Ubuntu. Yesterday's fix wasn't enough, I guess. I'm on it.

**Comment by solarsailer** [comment]
Okay, I got it to work on Ubuntu. Try the `0.6.4` and let me know if it's okay for you.

**Comment by nostyleguy** [comment]
Fixed!
Thanks for the prompt response!
